packages/react/src/analytics/analytics.ts

```diff
@@ -56,6 +66,47 @@ class AnalyticsWeb extends Analytics {
     }
   }
 
+  override async getTrackEventData(
+    type: TrackTypesValues,
+    event: string,
+    properties?: EventProperties,
+    eventContext?: ContextData,
+  ): Promise<EventData<TrackTypesValues>> {
+    this.processContext(eventContext);
+
+    return await super.getTrackEventData(type, event, properties, eventContext);
+  }
+
+  override context(): Promise<ContextData>;
+  override context(key: string): Promise<unknown>;
+  override async context(key?: string) {
+    if (key && key !== 'library.version') {
+      return super.context(key);
+    }
+
+    const context = await super.context();
+
+    this.processContext(context);
+
+    // if key are set and goes to this path, then key must be equal to analyticsVersion
+    return key ? get(context, key) : context;
+  }
+
+  processContext(context?: ContextData) {
+    if (context?.library) {
+      let analyticsVersions = '';
+
+      if (context?.library?.name && context?.library?.version) {
```

Comment:
Same here, `library` will always be set in this case so as name and version as well.

Reply:
Fixed